openshift/templates/deploy.dc.yaml

```diff
@@ -136,7 +122,7 @@ objects:
         app: ${NAME}-${SUFFIX}
       name: ${NAME}-${SUFFIX}-source
     spec:
-      host: ${NAME}-${SUFFIX}-source-${PROJECT_DEPLOY}.pathfinder.gov.bc.ca
+      host: ${NAME}-${SUFFIX}.pathfinder.gov.bc.ca
```

Comment:
This is for no mismatching on http and https routes, right?

Reply:
It's more for simplicity and consistent naming.  $SUFFIX contains either PR-### or PROD right now.  I dropped the source/docker part because only the source gets deployed, so it's redundant.